contracts/test/forkMainnet/AMMWrapperWithPath/Event.t.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import "contracts-test/forkMainnet/AMMWrapperWithPath/Setup.t.sol";
+import "contracts-test/utils/AMMUtil.sol";
+import "contracts/AMMQuoter.sol";
+
+contract TestAMMWrapperWithPathEvent is TestAMMWrapperWithPath {
+    event Swapped(AMMWrapperWithPath.TxMetaData, AMMLibEIP712.Order order);
+
+    AMMQuoter ammQuoter;
+
+    // Override the "beforeEach" block
+    function setUp() public override {
+        TestAMMWrapperWithPath.setUp();
+
+        ammQuoter = new AMMQuoter(IPermanentStorage(permanentStorage), address(weth));
```

Comment:
Could amm quoter intialization be inlined to the variable declaration like `AMMQuoter ammQuoter = new AMMQuoter(IPermanentStorage(permanentStorage), address(weth));`

Reply:
I tried to move the ammQuoter declaration into the setUp() function but compile shows the "DeclarationError: Undeclared identifier" error, I think it had a variable field consideration.

<details>
  <summary>( Please click here to expand )</summary>
    <pre><code>
      error[7576]: contracts/test/forkMainnet/AMMWrapperWithPath/Event.t.sol:30:37: DeclarationError: Undeclared identifier.
              uint256 expectedOutAmount = ammQuoter.getMakerOutAmountWithPath(
                                          ^-------^
    </code></pre>
</details>